### PR TITLE
Display annotation for iOS

### PR DESCRIPTION
**Pull request by vquach2404**

### CODE REVIEW

File: arcgis-ios-sdk-samples/Layers/Display annotation/README.metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "category": "Layers",
+    "description": "Display annotation from a feature service URL.",
+    "ignore": false,
+    "images": [
+        "DisplayAnnotation.png"
+    ],
+    "keywords": [
+        "Annotation",
```
Comment (philium) [inline arcgis-ios-sdk-samples/Layers/Display annotation/README.metadata.json]: I'm not sure the capitalization of the keywords matter. For all of the other metadata files, we just copied the keywords exactly as they appeared in the README.

File: arcgis-ios-sdk-samples/Layers/Display annotation/README.metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    ],
+    "redirect_from": [],
+    "relevant_apis": [
+        "AGSAnnotationLayer",
+        "AGSFeatureLayer"
+    ],
+    "snippets": [
+        "DisplayAnnotationViewController.swift"
+    ],
+    "title": "Display Annotation"
```
Comment (philium) [inline arcgis-ios-sdk-samples/Layers/Display annotation/README.metadata.json]: Capitalization does matter here. Esri's style is to only capitalize the first letter of the first word in a title, so this should be "Display annotation". 

```suggestion
    "title": "Display annotation"
```

That way it matches the capitalization of the other sample titles on the [developers website](https://developers.arcgis.com/ios/latest/swift/sample-code/). It looks like we weren't entirely consistent ("Symbolize Shapefile" should be "Symbolize shapefile", for example), but we tried our best.

**Comment by vquach2404** [comment]
Thanks Phil, I've addressed your comments. Ready for another review.

**Comment by vquach2404** [comment]
Hi @saratchandrakarumuri please review!

**Pull request merged**